src/Elcodi/Bundle/ProductBundle/Resources/config/services.yml

```diff
@@ -12,3 +12,4 @@ services:
         class: %elcodi.provider.product_collection.class%
         arguments:
             product_repository: @elcodi.repository.product
+            configuration_manager: @elcodi.manager.configuration
```

Comment:
I'd rather inject directly the configuration value with `@=elcodi_config('product.use_stock')`